specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-11-01/costmanagement.json

```diff
@@ -1098,6 +1098,232 @@
         }
       }
     },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/providers/Microsoft.CostManagement/getReservationDetailsReport": {
```

Comment:
this should be "createReservationDetailsReport" or "generateReservationDetailsReport".  "get" is an overloaded term and should not be used for POST actions.

Reply:
I agree with this comment but I need to discuss with PM on this. I will get back to this comment after the discussion

Reply:
updated to /generateReservationDetailsReport 👍